### DISCUSSION TITLE
Not able to add reviewers

### DISCUSSION THREAD

**Discussion by varunsaxena1332001** [community/community#42906] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi, 

I have created a PR from a fork 
But I am able to add reviewers to that PR

If someone know to resolve this issue then please reply as soon as possible.

**Comment by jfullstackdev** [comment]
there are many instances which I cannot guess but you can check this out 

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews
